js/app/dashboard/app.js

```diff
@@ -40,28 +40,35 @@ const drake = dragula({
 /**
  * Code to handle drag and drop from unassigned tasks to assigned
  */
-function configureOnDrop(allTasks, assignTasks) {
+function configureDrake(unassignedTasksContainer, allTasks, assignTasks) {
```

Comment:
Maybe it would be nice to have the code just above inside this function. And then sthg like afterwards : `let drake = configureDragula()`

Reply:
The code above doesn't need to know about Redux related stuff, this is why I did it like this.